### ISSUE TITLE
Let's check to see if WebSites are started in IIS

### ISSUE THREAD

**Issue by johage** [microsoft/CSS-Exchange#1094]
**Provide Version Number**
<Current>

**Describe the issue**
Run a check to see if FE/BE WebSites are started.

**Expected behavior**
Help us check corner cases where someone has Stopped FE and/or BE WebSite in IIS.

**Script Output**
If applicable, add the exception that wasn't handled.

**Additional context**
Import-Module WebAdministration
Get-WebSite | ?{$_.State -ne "Started"}

**Comment by dpaulson45** [comment]
@johage can you have this stopped and the application pools be running under that site?

**Comment by johage** [comment]
100% confirmed that the AppPools (since they live under the server object and not a specific WebSite) will stay Running even when the Site(s) are in a Stopped state.

**Comment by dpaulson45** [comment]
What was the issue that we were seeing? Anything else we need to collect from IIS Site information?

One of the cases I am currently working, might be good to know about the security information like require SSL for PowerShell vdir.

There is another issue #955 that is also asking for the certificate binding information.

I don't recall fully why I did decide to go with `appcmd.exe` for how I am currently getting this information, but I did run into an issue with trying to get the objects when running remotely in a test yesterday for some of the WebAdministration module cmdlets. 

![image](https://user-images.githubusercontent.com/22776718/172411538-f733807c-bee9-47a3-bbbc-ed8d270dbd9a.png)


I might be able to get it to work if I return a custom object vs the default object. And it looks like this should also work on Windows Server 2012, so this might be an option but not sure just yet.
